packages/webdriverio/src/utils.ts

```diff
@@ -183,6 +183,14 @@ export const axeFinishRun = (
   );
 };
 
+export const configureAxe = (client: Browser<'async'>): Promise<void> => {
```

Comment:
This name is a bit of a misnomer. Would be better as `configureAllowedOrigins`

```suggestion
export const configureAllowedOrigins = (client: Browser<'async'>): Promise<void> => {
```